pkg/apis/planetscale/v2/vitessshard_types.go

```diff
@@ -134,6 +134,10 @@ type VitessReplicationSpec struct {
 	//
 	// Default: Semi-sync is not enforced.
 	EnforceSemiSync bool `json:"enforceSemiSync,omitempty"`
+
+	// InitializeMaster can be used to enable or disable the operator initializing replication
+	// on a new or restored Vitess Shard object.
+	InitializeMaster bool `json:"initializeMaster,omitempty"`
```

Comment:
This has to be a pointer so we can tell the difference between true, false, and unset.

```suggestion
	InitializeMaster *bool `json:"initializeMaster,omitempty"`
```

Reply:
Done.